hooks/tk-maya_scene_operations.py

```diff
@@ -49,7 +49,9 @@ def scan_scene(self):
 
             # get the path and make it platform dependent
             # (maya uses C:/style/paths)
-            maya_path = ref.replace("/", os.path.sep)
+            maya_path = cmds.referenceQuery(
+                ref, filename=1, withoutCopyNumber=1
```

Comment:
Setting the arguments to `True` instead of `1` feels more comfortable to me. I didn't even know this worked.

Reply:
yeah fair point, I think I was just looking at examples and didn't think about it.